### ISSUE TITLE
Dataset as agent of CreateAction

### ISSUE THREAD

**Issue by dnlbauer** [nfdi4plants/arc-cwl-ro-crate-profile#15]
Hi,

While reading the draft, I noticed that the [CreateAction](https://schema.org/CreateAction) is intended to have the [Dataset](https://schema.org/Dataset) as [agent](https://schema.org/agent).

https://raw.githubusercontent.com/nfdi4plants/arc-cwl-ro-crate-profile/refs/heads/release/profile/arc_cwl_ro_crate.md?plain=1#L40

This seems a bit unusual, since schema.org typically uses agent to refer to a Person/Organization that executed the workflow. It's used to give provenance to the actor in contrast to the used dataset. Also not that the Workflow Run RO-Crate profile specifically allows Person/Organization.

I haven't found details for this choice of Dataset in the draft. I think it would be helpful to clarify why this choice has been made.

Thank you!

**Comment by HLWeil** [comment]
Yup this is definitely an error. On first glance, the correct property would be [subjectOf](https://schema.org/subjectOf), which is the inverse of `about`.
